pages/docs/widgets/variants/button.en.mdx

```diff
@@ -113,7 +126,18 @@ the [category table](#available-categories).
 If you do not supply the category ID, a dropdown will be rendered where the user must choose a category.
 
 ```html
-<a data-category-id="3">
+<a data-category-id="3"></a>
+```
+
+##### `subcategory-id` integer
```

Comment:
There are two types of widget, so this param needs to be added to `calculator.en.mdx` as well.